src/components/gallery/EventsGallery.tsx

```diff
@@ -0,0 +1,165 @@
+import Image from 'next/image';
+import Link from 'next/link';
+
+import ciga from '/public/img/events/ciga2023.png';
+import hof22 from '/public/img/events/hacktoberfest2022.png';
+import hof21 from '/public/img/events/hacktoberfest2021.png';
+import xtdk21 from '/public/img/events/xaralatourdk2021.png';
+
+import useTranslation from 'next-translate/useTranslation';
+
+const EventsGallery = () => {
+	const { t, lang } = useTranslation('gallery');
+	return (
+		<section className="">
+			<div className="mx-auto mb-16 max-w-3xl text-center">
+				<h2 className="text-3xl font-bold text-gray-900 sm:text-4xl">
+					{t('eventsGallery.firstHeading')}
+				</h2>
+
+				<p className="mt-4 text-gray-500 sm:text-xl">
+					{t('eventsGallery.secondHeading')}
+				</p>
+			</div>
+			<div className="grid grid-cols-1 gap-4 lg:grid-cols-2 lg:gap-16">
+				<div>
+					<article className="group">
+						<Link href="/">
```

Comment:
Do we need a link wrapping the images?

Reply:
The image must be clickable because it must be returned to another page with all the images related to the event.
I will implement this toom.